lib/regexp.js

```diff
@@ -1,7 +1,8 @@
 import { truncate } from './helpers'
 
 export default function inspectRegExp(value, options) {
-  const sourceLength = options.truncate - (2 + value.flags.length)
+  const flags = value.toString().match(/[gimsuy]*$/)[0]
```

Comment:
To prevent this from becoming a maintainence issue, as new flags are getting added, we could change this to be:

```suggestion
  const flags = value.toString().split('/')[2]
```

Reply:
Thanks for tip :+1: